### ISSUE TITLE
Desktop: Blank page on start

### ISSUE THREAD

**Issue by mkaz** [Automattic/wp-calypso#4293]
Once applying changes here https://github.com/Automattic/wp-desktop/pull/127 that fix Desktop builds, we are now seeing a blank page with W logo with the following console log errors:

```
Uncaught ReferenceError: preloadHub is not defined
desktop-app.js:140 Uncaught TypeError: window.AppBoot is not a function
```

**Comment by ockham** [comment]
@mkaz, see my updated PR description for #127 -- Automattic/wp-calypso#4148 fixes this.

**Comment by mkaz** [comment]
Fixed with #4148
